wsd/COOLWSD.cpp

```diff
@@ -472,7 +472,9 @@ void cleanupDocBrokers()
         // consider shutting down unused subforkits
         for (auto it = SubForKitProcs.begin(); it != SubForKitProcs.end(); )
         {
-            const std::string& configId = it->first;
+            // copy as it will be used after erase()
+            std::string configId = it->first;
```

Comment:
@caolanm could you please review this? Thanks.

Line 492 in the old version invalidates the iterator, then line 493 still uses the configId reference.

Reply:
ah yes, it was fine until I jammed in the UnitWSD callback